report.h

```diff
@@ -92,6 +92,12 @@ void report_counters();
 
 // Prinst voltage of motors
 void report_voltage();
+void calculate_motor_voltage();
+void calculate_force_voltage();
+// Variable for holding voltage value after ADC
+extern uint16_t voltage_result[VOLTAGE_SENSOR_COUNT];
```

Comment:
You might want to give this a better name.  Maybe analog_voltage_readings, analog_voltages, analog_voltage_values, andalog_read_storage

This is global storage for analogs so I would like the variable name to reflect that immediately in the variable name.